.eslintrc.js

```diff
@@ -13,5 +16,14 @@ module.exports = {
         "sourceType": "module"
     },
     "rules": {
+        "@typescript-eslint/no-explicit-any": "off",
+        "@typescript-eslint/no-unused-vars": "off",
+        "@typescript-eslint/no-namespace": "off",
+        "@typescript-eslint/ban-types":"off",
+        "no-case-declarations": "off",
+        "prefer-const": "off",
+        "no-empty": "off",
+        "no-var": "off",
```

Comment:
This one should be okay. I don't think `var` is used anywhere...

Reply:
There were some at `print-raw.ts`. I've removed the rule and changed occurrences of `var` to `let` in https://github.com/siefkenj/unified-latex/pull/42/commits/0ac4a930a24b8ee7f5530af36e2af2aa6964b01f.